src/KeyVault/KeyVault.Sdk/Customized/NetworkRuleSet.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Microsoft.Azure.Management.KeyVault.Models
+{
+    /// <summary>
+    /// A set of rules governing the network accessibility of a vault.
+    /// </summary>
+    public partial class NetworkRuleSet
+    {
+        public static readonly string BypassOptionAzureServices = "AzureServices";
+        public static readonly string DefaultActionOptionAllow = "Allow";
+
+        /// <summary>
+        /// An initialization method that performs custom operations like setting defaults
+        /// </summary>
+        partial void CustomInit()
+        {
+            Bypass = Bypass ?? NetworkRuleSet.BypassOptionAzureServices;
```

Comment:
@YanaXu Could you kindly add serval comments here to explain why we need to customize this model ? so that other developers can notice these points each version iteration.

Reply:
@BethanyZhou , it should be copied from sdk repository. It is there for couple years.